### PR TITLE
Add bssids to wifi_networks

### PR DESCRIPTION
**Pull request by obelisk**
For some reason BSSIDs were left out when I implemented this before. This adds them back as a comma delimited list. I'm not sure if having a new row for each is what we would like but if so I can do that too.

### CODE REVIEW

**Comment by osqueryer** [comment]
:-1: The commit 922a3da2a96c8c2b443c6f2d075f140d3286c81a (Job results: [6949](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6949/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 922a3da2a96c8c2b443c6f2d075f140d3286c81a (Job results: [6950](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6950/)) failed one or more tests (**Linux**).

File: osquery/tables/networking/darwin/wifi.mm [hunk 1]
```diff
@@ -66,6 +68,23 @@ Status getKnownNetworksKey(std::string& key) {
                                                        : INTEGER(0);
   } else if (CFGetTypeID(property) == CFStringGetTypeID()) {
     return stringFromCFString((CFStringRef)property);
+  } else if (CFGetTypeID(property) == CFArrayGetTypeID() && key == "bssids") {
+    std::string results;
+    // Fetch the first BSSID we can find
+    auto count = CFArrayGetCount((CFArrayRef)property);
```
Comment (muffins) [inline osquery/tables/networking/darwin/wifi.mm]: Does this and the one below need to be implicit casts?

File: osquery/tables/networking/darwin/wifi.mm [hunk 2]
```diff
@@ -66,6 +68,23 @@ Status getKnownNetworksKey(std::string& key) {
...
+    auto cfkey = CFStringCreateWithCString(
+        kCFAllocatorDefault, "LEAKY_AP_BSSID", kCFStringEncodingUTF8);
+
+    for (CFIndex i = 0; i < count; i++) {
+      auto dref =
+          (CFDictionaryRef)CFArrayGetValueAtIndex((CFArrayRef)property, i);
+      CFTypeRef value = nullptr;
+      if (CFDictionaryGetValueIfPresent(dref, cfkey, &value)) {
+        results += stringFromCFString((CFStringRef)value) + ", ";
```
Comment (theopolis) [inline osquery/tables/networking/darwin/wifi.mm]: Wait, doesn't this leave a trailing ", " at the end? You should prefer to create a set / vector and then `::join` them together with some glue (e.g., ", ").

**Comment by osqueryer** [comment]
:-1: The commit 33df3a0e3902d79ae49a171049f9bc6887f613c0 (Job results: [3998](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3998/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 33df3a0e3902d79ae49a171049f9bc6887f613c0 (Job results: [3999](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3999/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 33df3a0e3902d79ae49a171049f9bc6887f613c0 (Job results: [7104](https://jenkins.osquery.io/job/osqueryPullRequestLinux/7104/)) failed one or more tests (**Linux**).

**Comment by jkeljo** [comment]
> I'm not sure if having a new row for each is what we would like but if so I can do that too.

Another option is to put it as a JSON array and add the [SQLite JSON1 extension](https://www.sqlite.org/json1.html) to enable exploding it if desired

File: osquery/tables/networking/darwin/wifi.mm (hunk 1 above)
Comment (theopolis) [reply osquery/tables/networking/darwin/wifi.mm]: Yeah I does unfortunately.

**Comment by theopolis** [comment]
I think this approach is OK, up to you though.

**Comment by theopolis** [comment]
ping @obelisk!

**Comment by directionless** [comment]
Closing due to inactivity. Feel free to comment or re-open @obelisk

**Pull request closed without merging**